docs/tor.md

```diff
@@ -1,3 +1,15 @@
+# Tor and Specter Desktop
+
+When using Specter Desktop, there might be certain cases where you would want to make calls over the Tor network. The simplest way to setup Tor integration with Specter Desktop is to click on the Get Started button of the welcome page aand follow the setup wizard, which will install and configure Tor for you. You can also access it but going to Settings -> Tor tab, and clicking on Setup Tor.
+
+<img width="956" alt="Screen Shot 2021-04-24 at 10 42 08" src="https://user-images.githubusercontent.com/10667901/115953085-c4bd2f00-a4e9-11eb-8aa0-34e601681852.png">
```

Comment:
I'd prefer to put the screenshots into this repo and that link is not this repo, no?
Either here:
https://github.com/cryptoadvance/specter-desktop/tree/master/screenshots
or also in the docs directory?